releasenotes/README.md

```diff
@@ -26,12 +26,49 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.18.7 (Nov 22, 2023) [stable-rc]
+## 2.19.7 (Nov 29, 2023) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: If you specify Apex tests with the `--tests` flag of `project deploy start|validate`, the `--test-level` flag defaults to `RunSpecifiedTests`. (GitHub issue [#2396](https://github.com/forcedotcom/cli/issues/2396), plugin-deploy-retrieve PR [#812](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/812))
+
+    Many thanks to Christopher Bohlman ([cbohlman](https://github.com/cbohlman)) for contributing the code.  We put out a "Help Wanted" call, and he answered with aplomb. Community Collaboration in Action -- we love it!  
+
+* NEW: Manage source tracking in existing orgs with these two new commands:
+
+    * `org enable tracking`: Allow Salesforce CLI to track changes in your source files between your project and an org.
+    * `org disable tracking`: Prevent Salesforce CLI from tracking changes in your source files between your project and an org.
+ 
+    Enabling or disabling source tracking has no direct effect on the org, it affects only your local environment. Specifically, Salesforce CLI stores the setting in the org's local configuration file so that source tracking operations are, or aren't, executed when working with the org. Run the `org enable tracking` command on orgs that support source tracking, such as scratch orgs and Developer and Developer Pro sandboxes. The command returns an error if the org doesn't support tracking. Examples of orgs that don't support source tracking include Developer Edition orgs, production orgs, Partial Copy sandboxes, and Full sandboxes. 
+
+  This example shows how to enable source tracking on an org with alias `myscratch`:
+
+  ```bash
+  sf org enable tracking --target-org myscratch
```

Comment:
```suggestion
 Imagine you have a sandbox used for integration tests, and you want to deploy to it but not have to wait for tracking operations.  This example shows how to disable source tracking on an org with alias `mySandbox`:

  ```bash
  sf org disable tracking --target-org mySandbox
```

This will help avoid GitHub issue [#2495](https://github.com/forcedotcom/cli/issues/2495)